lib/mayaUsd/render/vp2RenderDelegate/mayaPrimCommon.h

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright 2021 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#ifndef HD_VP2_MAYA_PRIM_COMMON
+#define HD_VP2_MAYA_PRIM_COMMON
+
+#include "pxr/imaging/hd/types.h"
+#include "pxr/imaging/hd/changeTracker.h"
+
+PXR_NAMESPACE_OPEN_SCOPE
+
+struct MayaPrimCommon {
+    enum DirtyBits : HdDirtyBits
```

Comment:
I need to be able to mark the selection mode dirty for a prim from code where I don't know the prim's type, so make a common place for Vp2RenderDelegate prims to define dirty flags that they all share.

This can be the beginning the refactoring we are hoping to do for HdVP2Mesh and HdVP2BasisCurve.

I'm not sure MayaPrimCommon is the best name, maybe Vp2RenderDelegatePrimCommon? Any other suggestions?

Reply:
I think Vp2RenderDelegatePrimCommon would be too long without adding any important info. I like MayaPrimCommon, but since you ask for suggestions, I will dare going to the short extreme: Vp2Prim.